webapp/src/controllers/team.ts

```diff
@@ -82,6 +82,37 @@ export async function inviteTeamMemberApi(req, res) {
 	return dynamicResponse(req, res, 302, { redirect: `/${req.params.resourceSlug}/team` });
 }
 
+/**
+ * @api {delete} /forms/team/invite
+ * @apiName invite
+ * @apiGroup Team
+ *
+ * @apiParam {String} email Email of person to invite
+ */
+export async function deleteTeamMemberApi(req, res) {
+	const { memberId } = req.body;
+	//account with that memberId
+	const memberAccount = await getAccountById(memberId);
+	if(memberAccount) {	
+		const foundTeam = await getTeamById(req.params.resourceSlug);
+		const org = await getOrgById(foundTeam.orgId);
+		if(!org) {
+			return dynamicResponse(req, res, 403, { error: 'User org not found' });
+		} else {
+			if (foundTeam.members.some(m => m.equals(memberAccount._id))) {
+				return dynamicResponse(req, res, 403, { error: 'Cannot remove org user' });
+			}
+		}
+		if (!foundTeam.members.some(m => m.equals(memberAccount._id))) {
+			return dynamicResponse(req, res, 403, { error: 'User not found in your team' });
+		}
+		await removeTeamMember(req.params.resourceSlug, memberId.toString());
```

Comment:
If I'm being picky, you should just always call removeTeamMember, and use the "modifiedCount" returned from the db call to know whether somebody was removed